tests/testthat/test-profile_sector.R

```diff
@@ -271,3 +271,37 @@ test_that("outputs `profile_ranking_avg` at company level", {
   company <- unnest_company(out)
   expect_true(hasName(company, "reduction_targets_avg"))
 })
+
+test_that("given a product in 'ipr', when scenarios has also 'weo', then the product-result includes a 'weo*' `scenario` and maps to `NA` in `sector_profile` (#279, tiltIndicator#739)", {
```

Comment:
Here's the test for the specific case associated to this issue. Meeting this expectation breaks other tests so the job is incomplete.